packages/core/src/components/accordion/accordion.tsx

```diff
@@ -12,12 +12,17 @@ export class TdsAccordion {
   /** Set the variant of the Accordion. */
   @Prop() modeVariant: 'primary' | 'secondary' = null;
 
+  /** Removes the bottom border of the last Accordion item. */
+  @Prop() noLastBorder: boolean = false;
+
   render() {
     return (
       <Host
-        class={`tds-accordion ${
-          this.modeVariant !== null ? `tds-mode-variant-${this.modeVariant}` : ''
-        }`}
+        class={{
+          'tds-accordion': true,
+          [`tds-mode-variant-${this.modeVariant || ''}`]: Boolean(this.modeVariant),
```

Comment:
Not sure this change is needed? If it adds something we should update the same logic for all components, if not I'd say we skip it.

Reply:
Otherwise I will get a SonarCloud issue. Is that okay @mJarsater?

Reply:
Ah, alright! Is it a code duplication warning?

Reply:
I think it was a "no-nested-template-literals" when I was doing something like this:
`tds-accordion ${
          this.modeVariant !== null ? `tds-mode-variant-${this.modeVariant}` : ''
        } ${
          this.noLastBorder ? 'no-last-border' : ''
        }`